src/main/java/tech/jhipster/lite/generator/setup/license/infrastructure/primary/LicenseModuleConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package tech.jhipster.lite.generator.setup.license.infrastructure.primary;
+
+import static tech.jhipster.lite.generator.slug.domain.JHLiteFeatureSlug.LICENSE;
+import static tech.jhipster.lite.generator.slug.domain.JHLiteModuleSlug.APACHE_LICENSE;
+import static tech.jhipster.lite.generator.slug.domain.JHLiteModuleSlug.MIT_LICENSE;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import tech.jhipster.lite.generator.setup.license.application.LicenseModuleService;
+import tech.jhipster.lite.module.domain.resource.JHipsterModuleOrganization;
+import tech.jhipster.lite.module.domain.resource.JHipsterModuleResource;
+
+@Configuration
+public class LicenseModuleConfiguration {
+
+  @Bean
+  JHipsterModuleResource mitLicenseModule(LicenseModuleService license) {
+    return JHipsterModuleResource
+      .builder()
+      .slug(MIT_LICENSE)
+      .withoutProperties()
+      .apiDoc("MIT license", "Add MIT license file")
```

Comment:
The fist argument is the group, so I think it should be the same for all module resources. 
```suggestion
      .apiDoc("License", "Add MIT license file")
```